__tests__/app.test.js

```diff
@@ -204,3 +208,162 @@ describe('GET /api/reviews/:review_id/comments', () => {
+      .send(newComment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe(
+          'Type of the provided value does not match the type expected in the related database field'
+        );
+      });
+  });
+
+  it('404: responds with an error when the provided ID does not exist', () => {
+    const newComment = {
+      username: 'mallionaire',
+      body: 'I completely agree, this game is awesome!',
+    };
+
+    return request(app)
+      .post('/api/reviews/99999/comments')
+      .send(newComment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe('ID does not exist');
+      });
+  });
+});
+
+describe('PATCH /api/reviews/:review_id', () => {
+  it('200: returns updated review, with vote property incremented by value of inc_vote (a positive integer)', () => {
+    const testIncrementVote = { inc_vote: 5 };
+
+    return request(app)
+      .patch('/api/reviews/1')
+      .send(testIncrementVote)
+      .expect(200)
+      .then((response) => {
+        const { updatedReview } = response.body;
+
+        expect(updatedReview).toEqual({
+          review_id: 1,
+          title: 'Agricola',
+          review_body: 'Farmyard fun!',
+          designer: 'Uwe Rosenberg',
+          review_img_url:
+            'https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700',
+          votes: 6, // was 1, updated to 6
+          category: 'euro game',
+          owner: 'mallionaire',
+          created_at: '2021-01-18T10:00:20.514Z',
+        });
+      });
+  });
+
+  // NOTE: like Reddit's voting system, a negative integer is a valid value for votes
+  it('200: returns updated review, with vote property decremented when value of inc_vote is a negative integer', () => {
+    const testDecrementVote = { inc_vote: -2 };
+
+    return request(app)
+      .patch('/api/reviews/1')
+      .send(testDecrementVote)
+      .expect(200)
+      .then((response) => {
+        const { updatedReview } = response.body;
+
+        expect(updatedReview).toEqual({
+          review_id: 1,
+          title: 'Agricola',
+          review_body: 'Farmyard fun!',
+          designer: 'Uwe Rosenberg',
+          review_img_url:
+            'https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700',
+          votes: -1, // was 1, updated to -1
+          category: 'euro game',
+          owner: 'mallionaire',
+          created_at: '2021-01-18T10:00:20.514Z',
+        });
+      });
+  });
+
+  it('400: responds with an error when the increment value provided is not an integer', () => {
+    const testBadIncrement = { inc_vote: 1.5 };
+    return request(app)
+      .patch('/api/reviews/1')
+      .send(testBadIncrement)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe(
+          'Type of the provided value does not match the type expected in the related database field'
+        );
+      });
+  });
+});
```

Comment:
A couple of error tests to add here:
400 - invalid id 
404 - non-existent id
400 - missing body in the sent object